projects/observability/src/shared/components/explore-query-editor/group-by/explore-query-group-by-editor.component.ts

```diff
@@ -82,13 +86,15 @@ export class ExploreQueryGroupByEditorComponent implements OnChanges {
       merge(this.incomingGroupByExpressionSubject, this.groupByExpressionsToEmit)
     ]).pipe(map(optionsAndKey => this.resolveAttributeFromOptions(...optionsAndKey)));
 
-    this.groupByExpressionsToEmit
-      .pipe(
-        filter(expression => this.isValidExpressionToEmit(expression)),
-        debounceTime(500),
-        map(expression => (isEmpty(expression) ? undefined : omit(expression, 'metadata')))
-      )
-      .subscribe(this.groupByExpressionChange);
+    this.subscriptionLifecycle.add(
+      this.groupByExpressionsToEmit
```

Comment:
This one's good, it's a bug. But rather than wrapping this subscription in a lifecycle, can we complete the subject instead?